addons/rating/views/rating_templates.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
         <!-- External page : thanks message -->
-        <template id="rating_external_page_view" name="Rating Page Done">
+        <template id="rating_external_page_view" name="Thanks for your Feedback">
```

Comment:
Coucou @tde-banana-odoo ,

Just to be sure, you saw what we change in your module before I merge this PR. 🙂 😘 

Have a nice day! 🙂

Reply:
Thanks, indeed :)